src/pages/HomePage/components/BarTravel.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+import Button from '@components/Button/Button';
+import IcSearch from '@/assets/svg/IcSearch';
+import IcCheckin from '@assets/svg/IcCheckin';
+import IcCheckout from '@/assets/svg/IcCheckout';
+import IcPerson from '@assets/svg/IcPerson';
+import Spacing from './Spacing';
+import IcSeparator from '@/assets/svg/IcSeparator';
+import { containerStyle, searchIconStyle, iconStyle, buttonStyle, checkContainer, checkStyle, boxStyle, textStyle, dynamicTextStyle, checkLabelStyle, dateTextStyle, separatorStyle } from './BarTravel.style';
+
+const BarTravel = () => {
+  const [city, setCity] = useState<string | null>(null); 
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  useEffect(() => {
+    const queryParams = new URLSearchParams(location.search);
+    const cityFromURL = queryParams.get('city');
+    if (cityFromURL) {
+      setCity(cityFromURL);
+    }
+  }, [location]);
+
+  const handleSearchClick = () => {
+    if (city) {
+      navigate('/hotel-list'); 
+    }
+  };
+
+  return (
+    <div css={containerStyle}>
+      <button css={buttonStyle} onClick={() => navigate('/city')}>
+        <Spacing width={12} height={0} />
+        <IcSearch css={searchIconStyle} />
+        <Spacing width={12} height={0} />
+        {city ? city : "여행지/호텔명/프라이빗 하우스 검색"} 

```

Comment:
```suggestion
        {city || "여행지/호텔명/프라이빗 하우스 검색"} 
```
더 간결하게 표현할 수 있을 것 같아요 !

Reply:
최고 !!